mkdocs.yml

```diff
@@ -24,13 +24,13 @@ nav:
     - Illustrations:                      'fundamentals/illustrations.md'
     - Layouts:
       - Overview:                         'fundamentals/layouts/overview.md'
-      - Anatomy:                          'fundamentals/layouts/anatomy.md'
-      - Header:                           'fundamentals/layouts/header.md'
-      - Content:                          'fundamentals/layouts/content.md'
       - Breakpoints:                      'fundamentals/layouts/breakpoints.md'
-      - Grid:                             'fundamentals/layouts/grid.md'
+      - Flexbox grid:                     'fundamentals/layouts/grid.md'
       - Gutters:                          'fundamentals/layouts/gutters.md'
       - Spacing:                          'fundamentals/layouts/spacing.md'
+      - Anatomy:                          'fundamentals/layouts/anatomy.md'
```

Comment:
@panch1739  Not sure if we should put the highlevel chapters (anatomy, header, content) at the bottom? I think they are the more interesting ones and should remain at the top and the grid with all the details like gutters etc. should be at the bottom. The grid is more fundamental but not really required to read for a designer and acts more as a reference for devs when they implement something.

Reply:
@panch1739 Thanks for the adaptations 🙏 
Let's go with your suggestion for now until we have all the other chapters in and then we can adjust when needed.